apps/test-app/App.tsx

```diff
@@ -8,22 +8,44 @@ import {
   StatusText,
 } from "mocha-remote-react-native";
 
-import nodeAddonExamples from "@react-native-node-api/node-addon-examples";
+import { suites as nodeAddonExamplesSuites } from "@react-native-node-api/node-addon-examples";
 
-function loadTests() {
-  for (const [suiteName, examples] of Object.entries(nodeAddonExamples)) {
-    describe(suiteName, () => {
-      for (const [exampleName, requireExample] of Object.entries(examples)) {
-        it(exampleName, () => {
-          requireExample();
-        });
-      }
-    });
-  }
+function describeIf(
+  condition: boolean,
+  title: string,
+  fn: (this: Mocha.Suite) => void
+) {
+  return condition ? describe(title, fn) : describe.skip(title, fn);
+}
+
+type Context = {
+  allTests?: boolean;
+  nodeAddonExamples?: boolean;
+  ferricExample?: boolean;
+};
+
+function loadTests({
+  allTests = false,
+  nodeAddonExamples = allTests,
+  ferricExample = allTests,
```

Comment:
So by default, both `nodeAddonExamples` and `ferricExample` resolve to `false` and so both the "Node Addon Examples" and "ferric-example" tests get skipped? Is that the intention?

Reply:
I might still be debating this with myself 😊

On one hand it might be confusing to skip so many tests by default and on the other it does leave a potential to cut down on the time pre-building while bootstrapping and complexity (having to have the Rust toolchain installed) when someone wants to contribute to the project.

Looking back at this (I wrote it some weeks back), the optimization does seem a bit premature.